$MirOS: ports/net/djbdns/patches/patch-src_tai_now_c,v 1.1.201.1 2005/05/14 23:21:05 tg Exp $
--- src/tai_now.c.orig	Sun Feb 11 21:11:23 2001
+++ src/tai_now.c	Fri Feb 23 01:02:58 2007
@@ -1,7 +1,16 @@
+#include <sys/param.h>
+#if defined(MirBSD) && (MirBSD > 0x09AB)
+#include <sys/taitime.h>
+#endif
 #include <time.h>
 #include "tai.h"
 
 void tai_now(struct tai *t)
 {
+#ifndef __TAI64_BIAS
   tai_unix(t,time((time_t *) 0));
+#else
+  if (t != NULL)
+    t->x = tai_time(NULL);
+#endif /* __TAI64_BIAS */
 }
